src/buzzer-element.ts

```diff
@@ -0,0 +1,95 @@
+import { css, customElement, html, LitElement, property } from 'lit-element';
+
+/**
+ * Renders a piezo electric buzzer.
+ */
+@customElement('wokwi-buzzer')
+export class BuzzerElement extends LitElement {
+  /**
+   * Boolean representing if an electric signal is coming in the buzzer
+   * If true a music note will be displayed on top of the buzzer
+   */
+  @property() value = false;
+
+  static get styles() {
+    return css`
+      :host {
+        display: inline-block;
+      }
+
+      .buzzer-container {
+        display: flex;
+        flex-direction: column;
+        width: 75px;
+      }
+
+      .music-note {
+        position: relative;
+        animation-duration: 1.5s;
+        animation-name: animate-note;
+        animation-iteration-count: infinite;
+        animation-timing-function: linear;
+        transform: scale(1.5);
+        fill: blue;
+        left: 50px;
+        offset-path: path(
+          'm0 0c-0.9-0.92-1.8-1.8-2.4-2.8-0.56-0.92-0.78-1.8-0.58-2.8 0.2-0.92 0.82-1.8 1.6-2.8 0.81-0.92 1.8-1.8 2.6-2.8 0.81-0.92 1.4-1.8 1.6-2.8 0.2-0.92-0.02-1.8-0.58-2.8-0.56-0.92-1.5-1.8-2.4-2.8'
+        );
+        offset-rotate: 0deg;
+      }
+
+      @keyframes animate-note {
+        0% {
+          offset-distance: 0%;
+          opacity: 0;
+        }
+        10% {
+          offset-distance: 10%;
+          opacity: 1;
+        }
+        75% {
+          offset-distance: 75%;
+          opacity: 1;
+        }
+        100% {
+          offset-distance: 100%;
+          opacity: 0;
+        }
+      }
+    `;
+  }
+
+  render() {
+    const buzzerOn = this.value;
+    return html`
+      <div class="buzzer-container">
+        <svg
+          class="music-note"
+          style="visibility: ${buzzerOn ? '' : 'hidden'}"
+          xmlns="http://www.w3.org/2000/svg"
+          width="8"
+          height="8"
+          viewBox="0 0 8 8"
+        >
+          <path
+            d="M8 0c-5 0-6 1-6 1v4.09c-.15-.05-.33-.09-.5-.09-.83 0-1.5.67-1.5 1.5s.67 1.5 1.5 1.5 1.5-.67 1.5-1.5v-3.97c.73-.23 1.99-.44 4-.5v2.06c-.15-.05-.33-.09-.5-.09-.83 0-1.5.67-1.5 1.5s.67 1.5 1.5 1.5 1.5-.67 1.5-1.5v-5.5z"
```

Comment:
Not a deal breaker, but it'd be nice to combine this with the second SVG element. 
The `<path>` could be animated using CSS, similar to how you animate the `<svg>` element itself.

Reply:
Yes I'll do that.

Reply:
So I have tried and pushed. However i can't animate the note outside of the svg size. This means that i had to increase the svg height so that the animation fits inside it. This could lead to difficulties placing the buzzer in a webpage as there is blank area on top of it.

Reply:
Thanks for looking into that Glenn! Yes, that makes sense. It sounds like the original approach you took (with two SVG elements) makes more sense in this light.

Reply:
I reverted to 2 svg